TestFrame/Models/CountriesModels/CurrenciesModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+
+namespace TestFrame.Models.CountriesModels
+{
+    public class CurrenciesModel
+    {
+        [JsonExtensionData]
+        public JsonObject Currency { get; set; }
```

Comment:
not sure I get why this model is done like this

Reply:
It could be also rewritten like this:

[JsonExtensionData]
public Dictionary<string,object> Currency { get; set; }

From the research I made, [JsonExtensionData] suits for Dictionary like and JsonObject like properties. 
To explain why this approach, I will attach a print screen with the returned object when calling the API's endpoint:
![image](https://user-images.githubusercontent.com/72124598/230462910-4f90fa7f-6933-418d-99dd-5a8c5d00d12f.png)

Here, the object "currencies" has an object within itself as well - "RON". But this property's name ("RON") is dynamically changed depending on the parameter's value when calling the endpoint. 
For example: for Moldova, the object "currencies" would look like:
        "currencies": {
            "MD": {
                "name": "Moldovan leu",
                "symbol": "L"
            }
        }
So the property's name it's variable and dynamic. These solutions were the only ones I could find when deserializing the object, but I am open to other ideas. 
Hope this clarifies my intentions there.